File: SmartQuitIoT-Mobile/smartquitiot/lib/models/request/create_new_quit_plan_request.dart
class CreateNewQuitPlanRequest {
  final String startDate;
  final bool useNRT;
  final String quitPlanName;

  CreateNewQuitPlanRequest({
    required this.startDate,
    required this.useNRT,
    required this.quitPlanName,
  });

  Map<String, dynamic> toJson() => {
    'startDate': startDate,
    'useNRT': useNRT,
    'quitPlanName': quitPlanName,
  };
}

File: SmartQuitIoT-BE/src/main/java/com/smartquit/smartquitiot/dto/request/CreateNewQuitPlanRequest.java
package com.smartquit.smartquitiot.dto.request;

import java.time.LocalDate;
import lombok.*;
import lombok.experimental.FieldDefaults;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class CreateNewQuitPlanRequest {
  LocalDate startDate;
  boolean useNRT;
  String quitPlanName;
}
